2010-08-31  Mike Frysinger  <vapier@gentoo.org>

	* ports/sysdeps/unix/sysv/linux/alpha/sys/epoll.h (epoll_create2): Delete.
	(epoll_create1): Add.

---
 ports/sysdeps/unix/sysv/linux/alpha/sys/epoll.h |    9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

--- a/ports/sysdeps/unix/sysv/linux/alpha/sys/epoll.h
+++ b/ports/sysdeps/unix/sysv/linux/alpha/sys/epoll.h
@@ -1,4 +1,4 @@
-/* Copyright (C) 2002-2006, 2007, 2008 Free Software Foundation, Inc.
+/* Copyright (C) 2002-2008, 2010 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -31,7 +31,7 @@
 #endif
 
 
-/* Flags to be passed to epoll_create2.  */
+/* Flags to be passed to epoll_create1.  */
 enum
   {
     EPOLL_CLOEXEC = 010000000,
@@ -101,8 +101,9 @@
    returned by epoll_create() should be closed with close().  */
 extern int epoll_create (int __size) __THROW;
 
-/* Same as epoll_create but with an additional FLAGS parameter.  */
-extern int epoll_create2 (int __size, int __flags) __THROW;
+/* Same as epoll_create but with an FLAGS parameter.  The unused SIZE
+   parameter has been dropped.  */
+extern int epoll_create1 (int __flags) __THROW;
 
 
 /* Manipulate an epoll instance "epfd". Returns 0 in case of success,
